### DISCUSSION TITLE
DataGridTextColumn IsReadOnly=true should allow text selection and copy

### DISCUSSION THREAD

**Discussion by alexandrehtrb** [AvaloniaUI/Avalonia#13132] [Q&A] [answered]
Today, when a DataGridTextColumn is read-only, the cell's text can't be selected and copied; I think that would be desirable.

**Comment by timunie** [comment, marked as answer]
@alexandrehtrb copy & paste should work anyway. If not, probably try to specify `ClipboardContentBinding`. However, Text selection does not work. This is by design, as we have only a TextBlock rendered for being leightweight. 

I have two ideas for you:
1. Use a DataGridTemplateColumn and add a SelectableTextBlock as CellTemplate
2. Probably try if you can use a custom Style for DataGridCell using SelectableTextBlock instead. Haven't tried it on my own.

**Comment by timunie** [comment]
One more hint: Afaik, TreeDataGrid has cell selection. Maybe this is for you also a working solution?

Note: I see this as by design, unless I'm missing something critical (in that case add more info / screenshots). So I'll convert it into a discussion for now.
